packages/cli/src/oclif/commands/versions.js

```diff
@@ -23,6 +25,47 @@ class VersionCommand extends BaseCommand {
         'No versions to show. Try adding one with the `zapier push` command',
     });
 
+    this.logTable({
+      headers: [],
+      rows: [
+        {
+          version: `- ${colors.bold('Errors')}`,
+          platform_version:
+            'Issues that will prevent your integration from functioning properly. They block you from pushing.',
+        },
+        {
+          version: `- ${colors.bold('Publishing Tasks')}`,
+          platform_version:
+            'To-dos that must be addressed before your integration can be included in the App Directory. They block you from promoting and publishing.',
+        },
+        {
+          version: `- ${colors.bold('Warnings')}`,
+          platform_version:
+            "Issues and recommendations that need human reviews by Zapier before publishing your integration. They don't block.",
+        },
+      ],
+      hasBorder: false,
+      style: { head: [], 'padding-left': 0, 'padding-right': 0 },
+    });
+
+    this.logList([
```

Comment:
I think this call should be deleted! 😄

Reply:
You're right. I forgot to remove it that I used for validation during development.